theme/templates/accounts/profile.html

```diff
@@ -4,78 +4,516 @@
 {% block meta_title %}User profile for {{ u|best_name }}{% endblock %}
 
 {% block main %}
-{% with u.userprofile as profile %}
-<div class="row">
-<div class="col-md-3">
-    <h2>Profile information</h2>
+    {% with u.userprofile as profile %}
 
-    {% if profile.picture %}
-        <img class="img-thumbnail" src="{{ profile.picture.url }}" alt=""/>
-    {% endif %}
+        <div class="row" style="padding: 40px 0;">
+            {% if u.id == user.id %}
+                <div class="col-xs-12 col-sm-2 pull-right">
+                    <a id="btn-edit-profile" data-toggle="tooltip" data-placement="auto"
+                       title="Edit profile" href="{% url 'profile_update' %}"
+                       class="glyphicon glyphicon-pencil icon-button btn-edit"></a>
+                </div>
+            {% endif %}
 
-    <ul class="list-unstyled">
-        <li><strong>Name:</strong> {{ u.first_name }} {{ u.last_name }}</li>
+            <div {% if u.id == user.id %}class="col-xs-12 col-sm-10"{% else %}class="col-sm-12"{% endif %}>
+                {#        Profile picture       #}
+                {% if profile.picture and profile.picture.url %}
+                    <div style="background-image: url('{{ profile.picture.url }}'); float:left;"
+                         class="profile-pic round-image">
+                    </div>
+                {% endif %}
 
-        {% if profile.public %}
-            {% if profile.title %}
-            <li><strong>Title</strong> {{ profile.title }}</li>
-            {% endif %}
+                <div style="padding-top:10px;">
 
-            {% if profile.profession %}
-            <li><strong>Profession:</strong> {{ profile.profession }}</li>
-            {% endif %}
+                    {% if u.first_name or u.last_name %}
+                        <h2>{{ u.first_name }} {{ u.last_name }}</h2>
+                    {% else %}
+                        <h2>{{ u.username }}</h2>
+                    {% endif %}
 
-            {% if profile.subject_areas %}
-            <li><strong>Subject areas:</strong> {{ profile.subject_areas }}</li>
-            {% endif %}
+                    {% if profile.organization or profile.title %}
+                    <h4 class="text-muted">
+                        {% if profile.organization %}{{ profile.organization }}|{% endif %}
```

Comment:
you have a trailing `|` here to separate from the title even if the title is missing?